receiver/sapnetweaverreceiver/webservice.go

```diff
@@ -114,13 +121,37 @@ func (s *netweaverWebService) ABAPGetSystemWPTable() (*models.ABAPGetSystemWPTab
 	return response, nil
 }
 
-func (s *netweaverWebService) GetRequestLogFile() (*models.GetRequestLogFileResponse, error) {
-	request := &models.GetRequestLogFile{}
-	response := &models.GetRequestLogFileResponse{}
+func (s *netweaverWebService) OSExecute(command string) (*models.OSExecuteResponse, error) {
+	request := &models.OSExecute{
+		Command: command,
+	}
+	response := &models.OSExecuteResponse{}
 	err := s.client.Call("''", request, response)
 	if err != nil {
 		return nil, err
 	}
 
 	return response, nil
 }
+
+func (s *netweaverWebService) FindFile(args ...string) ([]string, error) {
+	resp, err := exec.Command("/usr/bin/find", args...).Output()
+	if err != nil {
+		return []string{}, err
+	}
+	// remove last new line
+	return strings.Split(string(strings.TrimRight(string(resp), "\n")), "\n"), nil
+}
+
+func (s *netweaverWebService) DpmonExecute(paths string) (string, error) {
+	cmd := exec.Command("bash", "-c", paths)
```

Comment:
Are we not able to get dpmon functioning with OSExecute?

Reply:
I am not able to use OSExecute here after testing.